### PR TITLE
Added links to the example projects to the top of each tutorial chapter

### PR DESCRIPTION
**Pull request by paradoxSW**

### CODE REVIEW

File: doc/basics/introduction/README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ This Source Code Form is subject to the terms of the Mozilla Public License, v.
 If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 -->
 # Introduction
+*You can find the example project [here](https://github.com/bmwcarit/ramses-composer-docs/tree/master/doc/basics/introduction).*
```
Comment (violinyanev) [inline doc/basics/introduction/README.md:11]: this is not really an example project, think we should remove the link

File: doc/basics/lua_syntax/README.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ If a copy of the MPL was not distributed with this file, You can obtain one at h
 -->
 
 # Lua Syntax
+*You can find the example project [here](https://github.com/bmwcarit/ramses-composer-docs/tree/master/doc/basics/lua_syntax).*
```
Comment (violinyanev) [inline doc/basics/lua_syntax/README.md:12]: same, no example project here

**Review by violinyanev — CHANGES_REQUESTED** [review]
Also, maybe worth adding a link to the trace player data?

**Comment by paradoxSW** [comment]
Fixed both links.
Not sure about the trace player, there is only the tracefile inside the traces folder, which is already linked at the end of the tutorial

**Review by violinyanev — APPROVED** [review]

**Pull request merged**